### ISSUE TITLE
Trial Balance without Group Totals

### ISSUE THREAD

**Issue by THEOSKaris** [frappe/erpnext#27131]
This Trial Balance will not

-  have group totals (i.e., the grouped totals highlighted in red).
- The Trial Balance will display only GL accounts and their balances.
- This saves a whole of time

 when Trial Balance is exported to Excel for independent confirmation of the balances.
![image](https://user-images.githubusercontent.com/67414153/130731071-d76d76a7-776e-43d8-a004-7c09ae8c52ec.png)

**Comment by kennethsequeira** [comment]
@THEOSKaris have you tried referring to the Trial Balance (Simple) report?

**Comment by THEOSKaris** [comment]
@kennethsequeira yes I have tried the Trial Balance (Simple) report. It does not display periodic account balances (as the standard Trial Balance does), rather it displays the Trial Balance per daily transactions.
![image](https://user-images.githubusercontent.com/67414153/130902170-662afea4-bd60-4230-86e6-e80d8593ef94.png)

**Comment by harshit-30** [comment]
@nabinhait can you check and reply to this?

**Comment by nabinhait** [comment]
@harshit-30 we can make it configurable, show group total based on the user's input, by default don't show.

**Comment by harshit-30** [comment]
@THEOSKaris I understand the changes you're requesting, but can you please explain why is it required? I am failing to understand the _why_ aspect of it.

**Comment by kennethsequeira** [comment]
@harshit-30 

The Trial Balance is supposed to be a simple listing of all the accounts with the balances. 

The grouping may or may not be useful given the context. 

For example, in the US users may prefer the report without grouping because that's how Quickbooks offers it and auditors tend to ask the report in the same format. 

Here's a sample from Quickbooks:

![image](https://user-images.githubusercontent.com/33246109/150780793-c9f6e7f8-39d9-40ef-9ac7-176e9d793c3d.png)

EDIT:

ERPNext's Trial Balance is closer to Tally's format:

![image](https://user-images.githubusercontent.com/33246109/150789236-431cd9f9-40aa-452b-81df-bf477777209a.png)

**Comment by THEOSKaris** [comment]
> @THEOSKaris I understand the changes you're requesting, but can you please explain why is it required? I am failing to understand the _why_ aspect of it.

@harshit-30 
Statutorily, the Trial Balance report format does not require grouping in Nigeria. Just like @kennethsequeira explained, auditors/accountants generally do not want the Trial Balance format with grouping, rather a simple listing of accounts, the balance of each account and the totals summing up all debit & credit balances.

**Comment by harshit-30** [comment]
@nabinhait instead of changing the Trial Balance Report, we can change the **Trial Balance Report (Simple)**. We can group each account's transactions and avoid repetition, same as @kennethsequeira shared in the previous comment.

Cc. @kennethsequeira @THEOSKaris

**Comment by pranaalee** [comment]
I feel, in the Trial Balance report or new one which could be a clone of it a column could be added with value of either yes or no in each row indicating if the account in that row is a Group or not. The users then can filter based on this value.

**Comment by THEOSKaris** [comment]
Thank you everyone for looking at this. I would like to follow up just know if there is update on it. Thank you.

**Comment by aakvatech** [comment]
> I feel, in the Trial Balance report or new one which could be a clone of it a column could be added with value of either yes or no in each row indicating if the account in that row is a Group or not. The users then can filter based on this value.

This is easily acheived and useful for the objective. Just Add a column from Account doctype and the field Is Group. This will help you filter in your export.

**Comment by SureshParvathareddy** [comment]
@aakvatech , But then bottom totals does not appear. The client always expects to see the TB report as simple as what @kennethsequeira has mentioned.

Does anyone know how to achieve this? Please guide me

**Comment by bernealmazan** [comment]
Export the Account table with the column "Is Group". Do vlookup in the Trial Balance exported file to show if it's "Is Group". What I did is insert in column A and the Excel formula looks like this = "=VLOOKUP(B8,Account!$A$2:$B$717,2,FALSE)". I added another sheet "Account" where I inserted the exported Account. You can then filter those "Is Group" = 0

**Comment by bernealmazan** [comment]
> [@aakvatech](https://github.com/aakvatech) , But then bottom totals does not appear. The client always expects to see the TB report as simple as what [@kennethsequeira](https://github.com/kennethsequeira) has mentioned.
> 
> Does anyone know how to achieve this? Please guide me

Unfortunately, the Show Totals is not in the Menu Items. As a workaround, You can just download to Excel and add a total line.

**Comment by nabinhait** [comment]
This is included in our current month's roadmap and is scheduled for implementation within the next couple of weeks.
